tests/integration/pages/desktop/dashboard.page.js

```diff
@@ -0,0 +1,22 @@
+/* global $ */
```

Comment:
Was this just to appease the ESLint gods, or was this needed for something else?

Reply:
For eslint. The ```$``` is needed for wdio though

Reply:
Ah, OK. You can also define [per-glob] globals in the ESLint config file (.eslintrc.js in root of project) instead of needing to define in each template.

For example, in the .eslintrc.js `overrides:` section, you can tweak the tests/**/*.js glob (around L32) and inject your `$`, `$$`, and `browser` globals there &mdash; if that is easier.

```diff
    {
      files: [
        'tests/**/*.js',
      ],
      env: {
        jest: true,
      },
+     globals: {
+       "$": "readonly",
+       "$$": "readonly",
+       "browser": "readonly",
+     },
    },
```

Reply:
Well that's pretty neat!